aces/Loris
# [Data Query Tool] Restored/updated DQT help text

Restoring help text for the DQT front end

## Brief summary of changes
help text file for the DQT module was accidentally deleted in PR #3390 - restoring it to display in the front end 
Some gentle updates and formatting improvements also included. 

#### Testing instructions (if applicable)

1. check out the branch
2. make sure the file is there
3.  Make sure the file shows up in the Help text dropdown for this module 

#### Links to related tickets (GitHub, Redmine, ...)
PR #3390
issue #5196 
https://github.com/aces/Loris/issues/5576
